packages/rum-recorder/src/domain/rrweb/record.ts

```diff
@@ -9,53 +9,11 @@ import { MutationController } from './mutation'
 let wrappedEmit!: (record: RawRecord, isCheckout?: boolean) => void
 
 export function record(options: RecordOptions = {}): RecordAPI {
-  const {
-    emit,
-    checkoutEveryNms,
-    checkoutEveryNth,
-    inlineStylesheet = true,
-    maskAllInputs,
-    maskInputOptions: maskInputOptionsArg,
-    slimDOMOptions: slimDOMOptionsArg,
-    maskInputFn,
-    hooks,
-    packFn,
-    sampling = {},
-    mousemoveWait,
-    recordCanvas = false,
-    collectFonts = false,
-  } = options
+  const { emit, slimDOMOptions: slimDOMOptionsArg } = options
```

Comment:
Note: I kept `slimDOMOptions` for now even if it is not used. I think it could be worth to enable it (to have lighter snapshots), we should discuss it and work on it in a future PR